CHANGELOG.rst

```diff
@@ -10,6 +10,14 @@ Added
 =====
 - Added a UI button for redeploying an EVC
 
+Changed
```

Comment:
Aldo, don't forget to also mention the v3 endpoints in the changelog, and also the DB scripts. Also, regarding the DB script, please map an issue similar [to this one](https://github.com/kytos-ng/kytos-ng.github.io/issues/60) as a reminder for the release notes

Reply:
Added to changelog